index.css

```diff
@@ -1,3 +1,45 @@
 html {
 	font-family: "ヒラギノ角ゴ Pro W3", "Hiragino Kaku Gothic Pro",Osaka, "メイリオ", Meiryo, "ＭＳ Ｐゴシック", "MS PGothic", sans-serif; 
 }
+
+.Grid {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	grid-gap: 5%;
+}
+
+/******** grid ***********/
+
+.Grid * {
```

Comment:
use the direct child selector instead .Grid > * - to avoid selecting ALL items inside grid.